packages/genesys-spark-components/src/components/stable/gux-toggle/gux-toggle.tsx

```diff
@@ -165,16 +165,15 @@ export class GuxToggle {
             'gux-disabled': this.disabled || this.loading
           }}
         >
-          <div class="gux-toggle-input">
+          <div class="gux-toggle-input" onClick={this.toggle.bind(this)}>
             <gux-toggle-slider
               checked={this.checked}
               disabled={this.disabled || this.loading}
               guxAriaLabel={this.getAriaLabel()}
-              onClick={this.toggle.bind(this)}
-              onKeyDown={this.onKeydown.bind(this)}
               labelId={
                 this.checkedLabel && this.uncheckedLabel ? this.labelId : ''
               }
+              onKeyDown={this.onKeydown.bind(this)}
```

Comment:
@321gillian a bit pedantic, but I would keep the 2 events together on the same element if possible just so we know that they are both there.

Reply:
@conor-darcy If the point is to allow more clickable area than just the toggle-slider (can be useful in some situations), I'm not sure if it's possible. The key down would be used with keyboard navigation and would need to be focused on the gux-toggle-slider to work afaik.

Reply:
@conor-darcy As @jordanstith15 said, in this case the keydown is linked to the focusable gux-toggle-slider. The click applies to both items in the gux-toggle-input.